index.html

```diff
@@ -1,313 +1,351 @@
               <ul class="category">
-                <li class="language1">html</li>
-                <li class="language2">bootstrap</li>
-                <li class="language3">Ruby</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
               </ul>
-              <button type="button">See Project</button>
             </div>
-          
-            <div class="work1" id="w2">
-              <h1>Profesional Art Printing Data</h1>
+            <button class="button" id="w1button">See Project</button>
+          </div>
+
+          <div id="w2" class="work1">
+            <div class="work1Discription" id="w2workDiscription">
+              <h2>
+                <span id="w2b">Profesional Art Printing Data</span>
+                <span id="w2d">Data Dashboard Healthcare</span>
+              </h2>
               <p>
-                A daily selection of privately personalized reads; no accounts or
-                sign-ups required. has been the industry's standard
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
               </p>
               <ul class="category">
-                <li class="language1">html</li>
-                <li class="language2">bootstrap</li>
-                <li class="language3">Ruby</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
               </ul>
-              <button type="button">See Project</button>
             </div>
-          
-            <div class="work1" id="w3">
-              <h1>Profesional Art Printing Data</h1>
+            <button class="button" id="w2button">See Project</button>
+          </div>
+
+          <div id="w3" class="work1">
+            <div class="work1Discription" id="w3work1Discription">
+              <h2>
+                <span id="w3b">Profesional Art Printing Data</span>
+                <span id="w3d">Website Protfolio </span>
+              </h2>
               <p>
-                A daily selection of privately personalized reads; no accounts or
-                sign-ups required. has been the industry's standard
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
               </p>
               <ul class="category">
-                <li class="language1">html</li>
-                <li class="language2">bootstrap</li>
-                <li class="language3">Ruby</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
               </ul>
-              <button type="button">See Project</button>
             </div>
-          
-            <div class="work1" id="w4">
-              <h1>Profesional Art Printing Data</h1>
+            <button class="button" id="w3button">See Project</button>
+          </div>
```

Comment:
- [ ] To match Figma design kindly fix these issues in this section : 

1. kindly add the correct margins & padding to the content of the cards shown in the image below 
2. kindly remove the border of the technologies squares/buttons

your design : 
![image](https://user-images.githubusercontent.com/89282221/194702413-ddc9ca43-1b98-42bf-ac70-387294d25248.png)


Figma Design : 
![image](https://user-images.githubusercontent.com/89282221/194702471-b1860d97-3e3a-4c51-b2f5-2831a1228cd1.png)